pkg/common/volume_operations_test.go

```diff
@@ -575,61 +617,75 @@ func TestVolumeOperationsImpl_ExpandVolume_DifferentStatuses(t *testing.T) {
 func TestVolumeOperationsImpl_ExpandVolume_Fail(t *testing.T) {
 	var (
 		svc      *VolumeOperationsImpl
-		err      error
 		capacity = int64(util.TBYTE)
 	)
 
 	volumeCR := testVolume1
 	svc = setupVOOperationsTest(t)
-	err = svc.k8sClient.CreateCR(testCtx, volumeCR.Spec.Id, &volumeCR)
+	volumeCR.ObjectMeta.CreationTimestamp = k8smetav1.Time{}
```

Comment:
Isn't better to use `volumeCR := testVolume1.DeepCopy()` everywhere despite of returning overridden fields back?

Reply:
👍